### ISSUE TITLE
Random vagrant time-outs when bringing the machines up

### ISSUE THREAD

**Issue by netoarmando** [freeipa/freeipa-pr-ci#265]
Vagrant fails when machines are being created:

```
2019-04-25 08:57:38,246    DEBUG  ==> client0: Starting domain.
2019-04-25 08:57:38,313    DEBUG  ==> controller: Starting domain.
2019-04-25 08:57:38,857    DEBUG  ==> master: Starting domain.
2019-04-25 08:57:38,932    DEBUG  ==> controller: Waiting for domain to get an IP address...
2019-04-25 08:57:39,165    DEBUG  ==> client0: Waiting for domain to get an IP address...
2019-04-25 08:57:39,178    DEBUG  ==> master: Waiting for domain to get an IP address...
2019-04-25 09:12:48,009    DEBUG  ==> replica0: Removing domain...
2019-04-25 09:12:49,304    DEBUG  ==> replica0: An error occurred. The error will be shown after all tasks complete.
2019-04-25 09:12:49,329    DEBUG  ==> client0: Removing domain...
2019-04-25 09:12:50,207    DEBUG  ==> controller: Removing domain...
2019-04-25 09:12:50,430    DEBUG  ==> controller: An error occurred. The error will be shown after all tasks complete.
2019-04-25 09:12:50,447    DEBUG  ==> client0: An error occurred. The error will be shown after all tasks complete.
2019-04-25 09:12:50,448    DEBUG  ==> master: Removing domain...
2019-04-25 09:12:50,666    DEBUG  ==> master: An error occurred. The error will be shown after all tasks complete.
2019-04-25 09:12:50,673    DEBUG  An error occurred while executing multiple actions in parallel.
2019-04-25 09:12:50,674    DEBUG  Any errors that occurred are shown below.
2019-04-25 09:12:50,675    DEBUG  
2019-04-25 09:12:50,675    DEBUG  An unexpected error occurred when executing the action on the
2019-04-25 09:12:50,676    DEBUG  'controller' machine. Please report this as a bug:
2019-04-25 09:12:50,676    DEBUG  
2019-04-25 09:12:50,677    DEBUG  The specified wait_for timeout (2 seconds) was exceeded
```
[Complete log](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/ffeacb1c-6735-11e9-867a-fa163ef3a1b7/runner.log.gz)

**Comment by dhnunes** [comment]
Closing as agreed

**Comment by netoarmando** [comment]
This happens to be a libvirt issue, more info here: https://github.com/vagrant-libvirt/vagrant-libvirt/issues/510#issuecomment-195326272

**Comment by netoarmando** [comment]
Related to https://github.com/freeipa/freeipa-pr-ci/issues/20.

**Comment by netoarmando** [comment]
Assuming this is a `won't fix` issue affecting vagrant/libvirt.
